File: php/Pc.php
<?php

include "Processor.php";
include "Disk.php";
include "Ram.php";

class Pc
{
    private $Processor_Object;
    private $Disk_Object;
    private $Ram_Object;
    private $TotalPc;

    function __construct($ProcessorX, $DiskX, $RamX, $Total)
    {
        $this->Processor_Object = $ProcessorX;
        $this->Disk_Object = $DiskX;
        $this->Ram_Object = $RamX;
        $this->TotalPc = $Total;
    }

    function setProcessorObject($ProcessorX)
    {
        $this->Processor_Object = $ProcessorX;
    }

    function getProcessorObject()
    {
        return $this->Processor_Object;
    }

    function setDiskObject($DiskX)
    {
        $this->Disk_Object = $DiskX;
    }

    function getDiskObject()
    {
        return $this->Disk_Object;
    }

    function setRamObject($RamX)
    {
        $this->Ram_Object = $RamX;
    }

    function getRamObject()
    {
        return $this->Ram_Object;
    }

    function setTotalPrice($Total)
    {
        $this->TotalPc = $Total;
    }

    function getTotalPrice()
    {
        return $this->TotalPc;
    }

    function print()
    {
        echo "Spesifikasi :<br/>";
        $this->Processor_Object->print();
        $this->Disk_Object->print();
        $this->Ram_Object->print();
        echo "Total (Rp) : " . $this->getTotalPrice() . "<br/>";
    }
}

File: python/Pc.py
from functools import total_ordering
from platform import processor
from Processor import Processor
from Ram import Ram
from Disk import Disk

class Pc:

    __Processor_Object = Processor()
    __Disk_Object = Disk()
    __Ram_Object = Ram()
    __TotalPc = 0

    def __init__(self, ProcessorX, DiskX, RamX, Total = 0):
        self.__Processor_Object = ProcessorX
        self.__Disk_Object = DiskX
        self.__Ram_Object = RamX
        self.__TotalPc = Total

    def setProcessorObject(self, ProcessorX):
        self.__Processor_Object = ProcessorX

    def getProcessorObject(self):
        return self.__Processor_Object
    
    def setDiskObject(self, DiskX):
        self.__Disk_Object = DiskX

    def getDiskObject(self):
        return self.__Disk_Object

    def setRamObject(self, RamX):
        self.__Ram_Object = RamX

    def getRamObject(self):
        return self.__Ram_Object

    def setTotalPrice(self, Total):
        self.__TotalPc = Total

    def getTotalPrice(self):
        return self.__TotalPc
    
    def print(self):
        print("Spesifikasi : ")
        self.__Processor_Object.print()
        self.__Disk_Object.print()
        self.__Ram_Object.print()
        print("Total (Rp) :" + str(self.getTotalPrice()))

    
